typescript/packages/browser/package.json

```
@@ -1,23 +1,23 @@
 {
     "name": "@0x/mesh-browser",
-    "version": "1.0.0",
+    "version": "8.2.0",
```

In the Git Flow model, code in the `development` is not associated with a specific version. Can we change this to "development" and then add in the correct version when we cut a new release?

```suggestion
    "version": "development",
```

---

I'm going to keep playing with this, but the current answer I'd give is "no" because it seems to break `yarn install` at the monorepo level.

---

If not `development`, perhaps just always keep it as `1.0.0`?

---

Agree `1.0.0` is the next best thing.